lib/dest/index.js

```diff
@@ -14,8 +15,9 @@ function dest(outFolder, opt) {
 
   var saveStream = pumpify.obj(
     prepare.dest(outFolder, opt),
+    makeDirs(opt),
```

Comment:
`makeDirs` can't come before `sourcemap` because `sourcemap` might emit files with a different dirpath than the original files.